docs/kubernetes/features.md

```diff
@@ -0,0 +1,114 @@
+# Features
+
+|Feature|Status|API Version|Example|Description|
+|---|---|---|---|---|
+|Managed Disks|Beta|`vlabs`|[kubernetes-vmas.json](../../examples/disks-managed/kubernetes-vmss.json)|[Description](#feat-managed-disks)|
+|Managed Identity|Alpha|`vlabs`|[kubernetes-msi.json](../../examples/managed-identity/kubernetes-msi.json)|[Description](#feat-kubernetes-msi)|
+|Calico Network Policy|Alpha|`vlabs`|[kubernetes-calico.json](../../examples/networkpolicy/kubernetes-calico.json)|[Description](#feat-calico)|
+
```

Comment:
Love having this product feature stability right in the docs instead of our emails... @anhowe and I can flesh this out over time.